pkg/controller/controller.go

```diff
@@ -0,0 +1,85 @@
+package controller
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/ablarry/golang-bootcamp-2020/pkg/model"
+	"github.com/ablarry/golang-bootcamp-2020/pkg/service"
+	"github.com/ablarry/golang-bootcamp-2020/pkg/utils/rest"
+)
+
+// Pokemon is a specific interface to manage http operations of Pokemon entities
+type Pokemon struct {
+	S service.Service
+}
+
+// GetList get list of pokemons
+func (c Pokemon) GetList(w http.ResponseWriter, r *http.Request) {
+	log.Println("Request GetList")
+	params := ParamsList{Size: 10, Paginator: 0}
+	if err := rest.DecodeURL(r, &params); err != nil {
+		rest.Handler(w, err)
+		return
+	}
+	pk, err := c.S.GetList(params.Paginator, params.Size)
+	if err != nil {
+		rest.Handler(w, err)
+		return
+	}
+	rest.JSON(w, pk, http.StatusOK)
+
```

Comment:
Let's remove this and all other trailing empty lines.

There are some linters that will be crying for stuff like that.
```suggestion
```